src/Common/CurrentMetrics.cpp

```diff
@@ -10,6 +10,7 @@
     M(Merge, "Number of executing background merges") \
     M(MergeParts, "Number of source parts participating in current background merges") \
     M(Move, "Number of currently executing moves") \
+    M(Export, "Number of currently executing exports") \
```

Comment:
i would probably add few counters to profile events smth like 
PartsExports (counter of successfull exports)
PartsExportFailures (counter of exports  with exception)
PartsExportDuplicated (when target exists)
PartsExportBytes (sum of the bytes written by exports) 
PartsExportTotalMilliseconds (overall time of exports) 

(Names - if you will have better ideas - feel free :) )